src/data/CardData.js

```diff
@@ -0,0 +1,32 @@
+import glasgow from "../images/glasgow.jpeg";
+import manchester from "../images/manchester.webp";
+import london from "../images/london.webp";
+
+export default function Data() {
```

Comment:
We capitalise functions when they are React _components_. React components return React elements that describe the UI. This function only returns an array literal.